### ISSUE TITLE
Upgrading to Helm v3

### ISSUE THREAD

**Issue by robasp** [rancher/rancher#46536]
**Rancher Server Setup**
- Rancher version: 2.8.1
- Installation option (Docker install/Helm Chart):
   - 1 Instance on Docker single-host
   - 1 Instance as Helm Chart on EKS

In the Rancher Release notes, it is noted that helm v2 is deprecated and will be removed in Rancher v2.9.
When i go to The About Page on the Rancher Web-Interface, it says that I am using Helm Version v2.16.8-rancher2.
My question is, how do i update the helm version? Is it done automatically with the Rancher Image Version?
Or what would i have to update manually before upgrading to 2.9.
It would be helpful to find information about this in the docs, but i couldn't find it.

Thanks for your help and best regards.

Robin

**Comment by mallardduck** [comment]
This may be a dashboard/visual issue more than a practical one. You can find our orgs fork of helm here: https://github.com/rancher/helm/releases

While the rancher images do still have V2 for compatibility, they do also have (and default AFAIK) to use v3 of helm. You can see the version included in 2.9 here: https://github.com/rancher/rancher/blob/da175cee0c1c62e6dbc90577bc548f6c9bd3c492/package/Dockerfile#L90

We pretty frequently update it to new helm versions, and then propagate that to our other components. For instance, our rancher/shell images (and other similar components) get updated to use new helm tags as they are released too.

**Comment by SamDecrock** [comment]
Thanks. I also noticed a higher version when checking the version via the web CLI.

**Comment by skanakal** [comment]
@rohitsakala The CATTLE_HELM_VERSION variable — this was completely [removed](https://github.com/rancher/rancher/pull/47661/commits/356f6c38e9c0a4f5dc111c3c1d193862fe70e585#diff-c55d1ee74f32cd4b9eeeceaec981a180f52e761acb158a5b0b0e088c7725c9caL60) from the backend starting with Rancher v2.11. Since then, Helm version is fetched from the dev environment. However, in upgraded clusters, it still seems to fall back to the old CATTLE_HELM_VERSION value, leading to inconsistencies.

I've opened a PR on the UI side to remove the Helm version display entirely. However, I received feedback that SETTING.VERSION_HELM should be updated with the current version instead.

If you're okay with reintroducing support for exposing the current Helm version (e.g., via SETTING.VERSION_HELM or similar), I'll open pr in rancher/rancher. Otherwise, we’ll proceed with removing it completely from the UI.

**Comment by jyoshnabonagirirp7** [comment]
Yes. reintroducing support for exposing the current Helm version is a good direction to go. Once you open the PR, I will review the solution.

**Comment by rohitsuse** [comment]
Yes. going through the route of reintroducing support for exposing the current Helm version. I will review your PR.

**Comment by gaktive** [comment]
Dashboard ticket closed since this is purely backend.

**Comment by rohitsuse** [comment]
QA Test Case 

- [x] Fresh Install Rancher : Need to check the HELM Version displayed is the same as the one in the rancher/shell image. Open kubectl shell in the UI and check the helm version. 
- [x] Same thing on Upgrade from a previous version to the version where this code lies.


cc @skanakal

**Comment by pankajsqe** [comment]
Test Pass : Rancher UI now showing correct helm version same as rancher shell. PFA

Details
Rancher : 2.13-head
Cluster : RKE2
Helm : v3.18.3-rancher1


- [x] Fresh Install Rancher : Need to check the HELM Version displayed is the same as the one in the rancher/shell image. Open kubectl shell in the UI and check the helm 

- [x] Same thing on Upgrade from a previous version to the version where this code lies.


<img width="1728" height="510" alt="Image" src="https://github.com/user-attachments/assets/4a90dadc-f979-4828-8b1d-cbebab267ab9" />
<img width="1728" height="952" alt="Image" src="https://github.com/user-attachments/assets/2302e0bb-7749-414b-8332-2a315914d2bb" />
